sympy/matrices/matrices.py

```diff
@@ -4010,12 +4010,12 @@ def a2idx(j, n=None):
         try:
             j = j.__index__()
         except AttributeError:
-            raise IndexError("Invalid index a[%r]" % (j,))
+            raise IndexError("Invalid index a[%r]" % j)
```

Comment:
I'm actually a little confused. I thought this function was in common. My guess is that the common functions have been moved to matrices. But whatever the case, I believe the first case was previously correct and the second case is correct after modifications.